### PR TITLE
 Fix static files on python3.7

### PR DESCRIPTION
**Pull request by pslacerda**

### CODE REVIEW

File: web/httpserver.py [hunk 1]
```diff
@@ -194,6 +194,7 @@ def __init__(self, environ, start_response):
         self.headers = []
         self.environ = environ
         self.start_response = start_response
+        self.directory = os.getcwd()
```
[the remark this answers is not in the document]
Comment (pslacerda) [reply web/httpserver.py]: If I remember correctly, it is related to the documentation specifying that the static files directory must be relative to the current working directory.

**Comment by cclauss** [comment]
@pslacerda. Can you please rebase?  Thanks much!

**Comment by pslacerda** [comment]
Hi @cclaus months later I finally resolved the conflict! I tried the "resolve conflict" button so it merged it. 

The test suite passed here for Python 3.7. I hope that this commit helps old projects keep going and newer ones integrates nice with old ones.

**Comment by pslacerda** [comment]
Are the 3.7 release on the roadmap?

**Pull request merged**